torch/autograd/grad_mode.py

```diff
@@ -14,7 +14,11 @@ class no_grad(_DecoratorContextManager):
     consumption for computations that would otherwise have `requires_grad=True`.
 
     In this mode, the result of every computation will have
-    `requires_grad=False`, even when the inputs have `requires_grad=True`.
+    `requires_grad=False`, even when the inputs have `requires_grad=True`,
+    UNLESS the result is a tensor created with `requires_grad=True`,
```

Comment:
nit: it might not be clear that this is applies to new tensor construction, since we're talking about doing "computation". Maybe we could explicitly point out the tensor construction?